cognite/neat/rules/exporter/_rules2dms.py

```diff
@@ -521,56 +498,186 @@ def find_existing_views(self, client: CogniteClient) -> set:
 
         return existing_views
 
-    def create_space(self, client: CogniteClient):
-        logging.info(f"Creating space {self.space}")
+    def find_existing_data_model(self, client: CogniteClient) -> DataModelId | None:
+        """Checks if the data model exists in CDF.
 
+        Args:
+            client: Cognite client.
+
+        Returns:
+             True if the data model exists, False otherwise.
+        """
+        if model := client.data_modeling.data_models.retrieve((self.space, self.external_id, self.version)):
+            cdf_data_model = model.latest_version()
+            return cdf_data_model.as_id()
+        return None
+
+    def to_cdf(
+        self,
+        client: CogniteClient,
+        components_to_create: set | None = None,
+        existing_component_handling: Literal["fail", "skip", "update"] = "fail",
+    ) -> None:
+        """Write the the data model to CDF.
+
+        Args:
+            client: Connected Cognite client.
+            components_to_create: Which components to create. Takes set
```

Comment:
So we are introducing two new concept to neat with this PR?
* Schema: View+container+data model
* Component: Which is an item in the schema?

Reply:
My proposal:

Schema contains following components:
- Spaces
- Containers
- Views
- Data Model

We had these from before, but it was not explicit, and all crammed under "data model".